### ISSUE TITLE
WAV file does not print the channels 

### ISSUE THREAD

**Issue by pmorelli92** [devsnd/tinytag#61]
Hello, as I describe when using any .wav file I get the following:

```
{'filesize': 35799444, 'album': None, 'albumartist': None, 'artist': None, 'audio_offset': None, 'bitrate': 1378.125, 'channels': None, 'comment': None, 'disc': None, 'disc_total': None, 'duration': 202.94401360544217, 'genre': None, 'samplerate': 44100, 'title': 'frenemy rem 7', 'track': None, 'track_total': None, 'year': None, 'audio_offest': 112}
```

Is there any way to fix this?

**Comment by devsnd** [comment]
Hey @pmorelli92 

You can try to edit this line:
https://github.com/devsnd/tinytag/blob/master/tinytag/tinytag.py#L827

and save the `channels` variable into `self.channels`. I'm not sure it will work as expected though.

**Comment by devsnd** [comment]
Hey @pmorelli92 

The issue is fixed in version 1.2.1, please update :dancer:
